src/translator/commands/sc_scn_tex2scs_translations.h

```diff
@@ -6,6 +6,10 @@
 using ScSCnTexTranslations = std::unordered_map<std::string, std::pair<std::string, std::string>>;
 
 static ScSCnTexTranslations const translations = {
+    {"База знаний Метасистемы OSTIS", { "knowledge_base_Metasystem", "sc_node_norole_relation" } },
+    {"Решатель задач Метасистемы OSTIS", { "problem_solver_Metasystem", "sc_node_norole_relation" } },
+    {"Пользовательский интерфейс Метасистемы OSTIS", { "user_interface_Metasystem", "sc_node_norole_relation" } },
+    {"Множество подсистем Метасистемы OSTIS", { "concept_subsystems_set_Metasystem", "sc_node_norole_relation" } },
```

Comment:
```suggestion
    {"База знаний Метасистемы OSTIS", { "knowledge_base_Metasystem", "sc_node_norole_relation" } },
    {"Решатель задач Метасистемы OSTIS", { "problem_solver_Metasystem", "sc_node_norole_relation" } },
    {"Пользовательский интерфейс Метасистемы OSTIS", { "user_interface_Metasystem", "sc_node_norole_relation" } },
    {"Подсистемы Метасистемы OSTIS", { "metasystem_subsystems", "sc_node_norole_relation" } },
```
Я бы сделал так и заодно надо в стандарте проверить